client/js/screens/Profile/Profile.js

```diff
@@ -1,10 +1,36 @@
 import React from "react";
-import { Text, View } from "react-native";
+import { Image, Text, TouchableOpacity, ScrollView, View } from "react-native";
 
-const Profile = ({ params }) => (
-  <View>
-    <Text>This is the Profile Screen</Text>
-  </View>
-);
+const Profile = ({ user }) => {
+  console.log("user", user);
+  return (
+    <ScrollView>
+      <View>
+        <Text>My Profile</Text>
+        <TouchableOpacity>
+          <Text>Edit</Text>
+        </TouchableOpacity>
+      </View>
+      <View>
+        <Image
+          style={{ width: 200, height: 200, borderRadius: 100 }}
```

Comment:
Move this to a style object

Reply:
Will move it to the style.js in the next step